raw_data/preprocessed/europe/france_french.txt

```diff
@@ -0,0 +1,97 @@
+
+(30) Les procès-verbaux dressés par les fonctionnaires et agents désignés à l'article 29 ci-dessus font foi jusqu'à preuve du contraire. Ils sont remis ou envoyés par lettre recommandée, directement au procureur de la République. Cette remise ou cet envoi doit avoir lieu, à peine de nullité, cinq jours francs après celui où l'infraction a été constatée. Les règles de procédure pénale édictées par les articles 17 à 21 du décret du 9 janvier 1852 précité sont applicables en cas d'infractions commises sur le domaine public maritime ou dans les eaux territoriales.
+
+(31) Les fonctionnaires et agents désignés à l'article 29 ci-dessus sont habilités, dans l'exercice de leurs fonctions, à visiter les réserves naturelles en vue de s'assurer du respect des règles auxquelles elles sont soumises et d'y constater toute infraction. Quiconque mettra ces fonctionnaires ou agents dans l'impossibilité d'accomplir leurs fonctions, notamment en leur refusant l'entrée d'une réserve naturelle, sera passible des peines prévues à l'article 32 ci-après sans préjudice, le cas échéant, des peines prévues par les articles 209 et suivants du code pénal.
+
+(32) Sont punies d'une amende de 2.000 à 40.000 F les infractions aux dispositions des articles 3, 4, 5, 6, 7, 21, 22, 23, 24 et 31 de la présente loi. En cas de récidive, l'amende pourra être portée à 80.000 F. En outre, les infractions aux dispositions de l'article 3 sont passibles des sanctions prévues aux premier et quatrième alinéas de l'article 379 du code rural. Les agents chargés de constater ces infractions peuvent procéder à la saisie des dépouilles.
+
+(33) Les articles 529 à 530-1 du code de procédure pénale sont applicables en matière d'infraction à la législation ou à la réglementation des parcs nationaux lorsque l'infraction commise est punie d'une amende pénale dont le montant n'excède pas un maximum fixé par décret en Conseil d'État. Toutefois, pour ces infractions, l'amende forfaitaire ne peut être acquittée qu'au moyen d'un timbre-amende.
```

Comment:
"530-1" appears to be "530·1" in the legislation. Not sure if this change is intentional or not. Also, I wonder if they are just referring to article 530 subsection 1 through this notation.

Reply:
The text selection software on my computer detected a hyphen; I did not intentionally replace a bullet. To be honest, I am not so sure that the separator should be a bullet. If you look at the text of article 34 in the legislation, for instance, the separator between "67" and "1174" in the number of one of the laws that is referenced also looks very similar to a bullet, but it is undoubtedly a hyphen.

Reply:
Thanks for looking into it. I agree it's probably not a big deal and having it as a hyphen does seem to be consistent with other legislation.